decompiler/util/DataParser.cpp

```diff
@@ -3,6 +3,10 @@
 #include "DataParser.h"
 #include "third-party/fmt/core.h"
 
+#ifdef __APPLE__
+#include <unordered_map>
```

Comment:
you don't need the `__APPLE__` around this - I should have put the `#include <unordered_map>` but it happened not to matter on linux/windows because some other header brought it in.